replay_build_scripts/update-repo.js

```diff
@@ -0,0 +1,3 @@
+const { updateRepo } = require("./updateRepo");
```

Comment:
I assume this file exists so that it can be easily invoked as a script?  A little comment explaining that might be nice.

Reply:
Yeah exactly, though that may not be necessary I'm not sure. I'll definitely add a comment though, thanks